plugin/src/compilation/decompiler.ml

```diff
@@ -52,7 +60,17 @@ let get_pushed_names env =
          | Anonymous ->
             failwith "Unexpected Anonymous in get_pushed_names."
          | Name n -> n) names)
-    
+
+(* Functions used with the rewrite tactic. *)
+let is_rewrite_r trm : bool =
+  let eq_term = equal trm in
+  eq_term eq_ind_r || eq_term eq_rec_r (* || eq_term eq_rect_r *)
+  
+(* Rewrite functions using "<-". *)
```

Comment:
This mapping might be reversed when you get to "rewrite in." We can handle that later, though.

Reply:
Yup you’re right it’s reversed in the terms I’ve looked at. Was thinking of having a separate IR constructor for RewriteIn.

Reply:
OK, feel free to punt on this until you implement RewriteIn